mac/Keyman4MacIM/Keyman4MacIM/Privacy/PrivacyConsent.m

```diff
@@ -0,0 +1,244 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ * 
+ * PrivacyConsent.m
+ * TestInputMethod2
+ * 
+ * Created by Shawn Schantz on 2022-09-22.
+ * 
+ * Used to determine if the user has provided consent to the services it needs
```

Comment:
Is this the same comment as in the .h? I think we probably only want it once (as they'll inevitably diverge in the future otherwise.)

I guess probably better to have this kind of info in the implementation -- .m file -- so you may want to take my formatting suggestion and apply it here...